alibi/datasets.py

```diff
@@ -29,6 +31,52 @@
 MOVIESENTIMENT_URLS = ['https://storage.googleapis.com/seldon-datasets/sentence_polarity_v1/rt-polaritydata.tar.gz',
                        'http://www.cs.cornell.edu/People/pabo/movie-review-data/rt-polaritydata.tar.gz']
 
+IMAGENET_URLS = ['https://storage.googleapis.com/seldon-datasets/imagenet10/imagenet10.joblib']
+
+
+def fetch_imagenet_10(url_id: int = 0) -> Union[Dict, Tuple[np.ndarray, np.ndarray]]:
+    """
+    Sample dataset extracted from imagenet in a dictionary format.
+    The train set contains 1000 random samples, 100 for each of the following 10 selected classes:
+
+    'stingray'
+    'trilobite'
+    'centipede'
+    'slug'
+    'snail'
+    'Rhodesian ridgeback'
+    'beagle'
+    'golden retriever'
+    'sea lion'
+    'espresso'
+
+    The test set contains 50 random samples, 5 for each of the classes above.
+
+    Parameters
+    ----------
+    url_id
+        Index specifying which URL to use for downloading
+
+    Returns
+    -------
+    Dictionary with the follwoing keys
+        `trainset`: train set tuple (X_train, y_train)
+        `testset`: test set tuple (X_test, y_test)
+        `int_to_str_labels`: map from target to target name
+        `str_to_int_labels`: map from target name to target
+
```

Comment:
Not sure if those render properly. Check the [CONTRIBUTING.md](https://github.com/SeldonIO/alibi/blob/master/CONTRIBUTING.md) for an example

Reply:
Docstring currently not showing at all: https://seldon--650.org.readthedocs.build/projects/alibi/en/650/api/alibi.datasets.html

Reply:
Fixed it. Function needs to be added to __all__